wiki/misconceptions.md

```diff
@@ -0,0 +1,187 @@
+# Misconceptions
+
+{title candidates: Misconceptions, Rebuttals, Common criticism}
+
+Misconceptions and common arguments against on-chain governance, coin voting and other aspects of Decred system.
+
+## Coin voting is plutocracy
+ 
+Counterarguments:
+
+- Company shareholder governance is also plutocracy, but nobody complains about that.
```

Comment:
People do complain about it, and they're right to because it's not good. I would move this down and say something like "The concept of governance by the people who share a stake in a common enterprise is familiar from the governance of corporations, although it works in a different way in that context".

Reply:
"People do complain about it, and they're right to because it's not good." - is this about shareholder governance? What do people complain about?